src/wp-includes/template.php

```diff
@@ -457,6 +457,7 @@ function get_privacy_policy_template() {
  * @since 1.5.0
  * @since 4.7.0 The decoded form of `page-{page_name}.php` was added to the top of the
  *              template hierarchy when the page name contains multibyte characters.
+ * @since 6.0.0 For block themes, use locate_block_template function instead.
```

Comment:
Instead of using the `since` mention, I think we should place this recommendation in the long description of the function :)

Reply:
Requested changes are made and pushed.